src/bin/pg_send_cancellation/Makefile

```
@@ -10,7 +10,11 @@ PG_LDFLAGS += $(LDFLAGS)
 include $(citus_top_builddir)/Makefile.global
 
 # We reuse all the Citus flags (incl. security flags), but we are building a program not a shared library
-override CFLAGS := $(filter-out -shared,$(CFLAGS))
+# We sometimes build Citus with a newer version of gcc than Postgres was built
+# with and this breaks LTO (link-time optimization). Even if disabling it can
+# have some perf impact this is ok because pg_send_cancellation is only used
+# for tests anyway.
+override CFLAGS := $(filter-out -shared,$(CFLAGS)) -fno-lto
```

```suggestion
override CFLAGS := $(filter-out -shared, $(CFLAGS)) -fno-lto
```

A single space makes it easier to read and understand.